packages/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.0
+
+* **Breaking Change** Change iOS image stream format to `ImageFormatGroup.bgra8888`.
```

Comment:
How and why is this breaking? If I'm a dev that would be broken by upgrading to this version, how can I know? If I already installed this version and I'm currently broken, is there anything you can tell me here on how to fix the breakage?

Reply:
This is a breaking change because the original format of the image stream buffer was `kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange` and now it is `kCVPixelFormatType_32BGRA`. This will most likely be a breaking change for a very small number of people, if any. I added this detail to the changelog.

If a developer was expecting `kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange` the best solution for them would be to use a conversion library on the platform side. Do you think I should link the iOS `v_image` library in the changelog?